src/sfizz/EQPool.cpp

```diff
@@ -108,9 +108,9 @@ sfz::EQPool::EQPool(const MidiState& state, int numEQs)
 
 sfz::EQHolderPtr sfz::EQPool::getEQ(const EQDescription& description, unsigned numChannels, float velocity)
 {
-    AtomicGuard guard { givingOutEQs };
-    if (!canGiveOutEQs)
+    if (!eqGuard.try_lock())
         return {};
+    DEFER(eqGuard.unlock());
```

Comment:
Try-locks are doable simply with use of `unique_lock`.
```
std::unique_lock<std::mutex> lock(mutex, std::try_to_lock);
if (!lock.owns_lock()) [...]
```

Reply:
Man you're killing my defer macro :sob:
Thanks for the tip, I changed the locks.